extensions/interactions/MathExpressionInput/directives/MathExpressionInputRulesService.ts

```diff
@@ -19,6 +19,22 @@
 oppia.factory('MathExpressionInputRulesService', [function() {
   return {
     IsMathematicallyEquivalentTo: function(answer, inputs) {
+      try {
+        MathExpression.fromLatex(answer.latex);
+      } catch (e) {
+        throw Error(
```

Comment:
Will this construct cause the stacktrace of e (the part inside MathExpression) to be lost?

Reply:
Actually, I saw a pattern in @bansalnitish's PR for this. We might as well use it?

https://github.com/oppia/oppia/pull/6808/files#diff-06e549f3712a3cf2db2270030f7c4c33R35